pytorch_lightning/plugins/precision/hpu.py

```diff
@@ -22,9 +22,24 @@
 
 
 class HPUPrecisionPlugin(PrecisionPlugin):
-    """Plugin that enables bfloats/floats on HPUs."""
+    """Plugin that enables bfloat/half support on HPUs.
 
-    def __init__(self, precision: Union[str, int], hmp_params: Optional[Sequence[Any]] = None) -> None:
+    Args:
+        precision: The precision to use.
+        opt_level: Choose optimization level for hmp.
+        bf16_file_path: Path to bf16 ops list in hmp O1 mode.
+        fp32_file_path: Path to fp32 ops list in hmp O1 mode.
+        verbose: Enable verbose mode for hmp.
+    """
+
+    def __init__(
+        self,
+        precision: Union[str, int],
+        opt_level: str = "O2",
```

Comment:
on the trainer this is `amp_level` - this can be confusing for what terminology to use.
does setting `amp_level` automatically set the `opt_level` ?

Reply:
Its confusing to mix AMP param with HMP.

Reply:
> >does setting amp_level automatically set the opt_level
> 
> I have updated to do that.

I have removed the automatic setting, let's not mix AMP param with HMP